feat: should be able to see the balance (#34)
Coderockr/backend-test: README.md

    2. An investment should not be or become negative.
2. __View__ of an investment with its initial amount and expected balance.
    1. Expected balance should be the sum of the invested amount and the [gains][].

>     2. If a Investment was already withdrawn then the expected balance must be calculed until the withdrawal date instead of today.

Reviewer: ```suggestion
    2. If an investment was already withdrawn then the balance must reflect the gains of that investment
```
